tests/sympc/tensor/share_tensor_test.py

```diff
@@ -0,0 +1,188 @@
+import pytest
+import syft as sy
+import torch
+
+from sympc.session import Session
+from sympc.tensor import ShareTensor
+
+
+@pytest.fixture
+def clients():
```

Comment:
Maybe this fixture should be in tests/conftest.py like in https://github.com/OpenMined/PySyft/blob/syft_0.2.x/test/conftest.py

Reply:
Yes will do it!